cardano-cli/test/cardano-cli-golden/Test/Golden/Key/NonExtendedKey.hs

```diff
@@ -83,12 +77,30 @@ hprop_golden_extended_payment_vkey_to_non_extended_vkey =
     goldenFile <-  H.note "test/cardano-cli-golden/files/golden/key/non-extended-keys/non-extended-payment.vkey"
     outFp <- H.note $ tempDir </> "non-extended-payment.vkey"
 
-    H.assertFilesExist [extendedKeyFile]
-
     void $ execCardanoCLI
       [ "conway", "key", "non-extended-key"
       , "--extended-verification-key-file", extendedKeyFile
       , "--verification-key-file", outFp
       ]
 
     H.diffFileVsGoldenFile outFp goldenFile
+
+-- | Test that converting a CC extended verification key yields the expected result.
+-- | Execute me with:
+-- @cabal test cardano-cli-golden --test-options '-p "/golden extended cc vkey to non extended vkey/"'@
```

Comment:
I personally like those comments, because they allow me to run one test without having to think about the complex shell command to enter. So far people received those new comments favorably, you're the first one to remark them :slightly_smiling_face: So I'm tempted to leave it here :innocent: